extensions/wikia/EditPreview/EditPreview.setup.php

```diff
@@ -21,16 +20,4 @@
 ];
 
 // messages
-$wgExtensionMessagesFiles[ 'EditPreview' ] = $dir . 'EditPreview.i18n.php';
```

Comment:
The messages are still used by the EditPageLayout JSMessages package (https://github.com/Wikia/app/blob/dev/extensions/wikia/EditPageLayout/EditPageLayout_setup.php#L84 ), only the EditPreview JSMessages package is unused